v2/pkg/subscraping/sources/quake/quake.go

```
@@ -56,9 +56,10 @@ func (s *Source) Run(ctx context.Context, domain string, session *subscraping.Se
 			s.skipped = true
 			return
 		}
-
+		//翻页设计
+		var pages = 1
```

golangci-lint (1.64.8)

ineffectual assignment to pages

(ineffassign)